tests/phpunit/tests/Checker/Checks/I18n_Usage_Check_Tests.php

```diff
@@ -58,8 +58,8 @@ public function test_run_without_default_textdomain() {
 
 		// Explicitly using the 'default' text domain is a warning, omitting a text domain is an error.
 		$this->assertNotEmpty( $check_result->get_errors() );
-		$this->assertNotEmpty( $check_result->get_warnings() );
-		$this->assertEquals( 1, $check_result->get_error_count() );
-		$this->assertEquals( 1, $check_result->get_warning_count() );
+		$this->assertEmpty( $check_result->get_warnings() );
+		$this->assertEquals( 2, $check_result->get_error_count() );
+		$this->assertEquals( 0, $check_result->get_warning_count() );
```

Comment:
Why did the error count change? 🤔

Reply:
Yah, above changes should not hit any tests I believe.

Reply:
Ah, `'default'` is not converted to `&#039;default&#039;` in the message.

Reply:
Rather than updating tests, it should be fixed here instead https://github.com/WordPress/plugin-check/blob/trunk/includes/Checker/Checks/General/I18n_Usage_Check.php#L102

Reply:
yeah, ok

Reply:
done